src/coreclr/jit/emitxarch.cpp

```diff
@@ -2631,8 +2628,7 @@ inline size_t insCodeRR(instruction ins)
 }
 
 // clang-format off
-const static
-size_t          insCodesRM[] =
+const static size_t insCodesRM[] =
```

Comment:
There is only one instruction larger than `uint32_t` and that's `movsxd`. This is because it hard encodes the `REX.W` bit

We should fix that to add the REX.W bit normally.

Reply:
Fixing this would save us `~2920` more bytes